packages/bls12-381/bls12-381.6.0.1/opam

```diff
@@ -0,0 +1,46 @@
+opam-version: "2.0"
+synopsis: """\
+Implementation of BLS12-381 and some cryptographic primitives built
+on top of it"""
+maintainer: "Danny Willems <be.danny.willems@gmail.com>"
+authors: "Danny Willems <be.danny.willems@gmail.com>"
+license: "MIT"
+homepage:
+  "https://nomadic-labs.gitlab.io/cryptography/ocaml-bls12-381/bls12-381/"
+bug-reports:
+  "https://gitlab.com/nomadic-labs/cryptography/ocaml-bls12-381/issues"
+depends: [
+  "dune" {>= "3.0"}
+  "ocaml" {>= "4.08"}
+  "ff-sig" {>= "0.6.1" & < "0.7.0"}
+  "zarith" {>= "1.10" & < "2.0"}
+  "zarith_stubs_js" {with-test}
+  "hex" {>= "1.3.0"}
+  "alcotest" {with-test}
+  "integers"
+  "integers_stubs_js" {with-test}
+  "ff-pbt" {>= "0.6.0" & < "0.7.0"}
+  "odoc" {with-doc}
+]
```

Comment:
The `depends` field should have a `conf-emcc { with-test }`. However, there is now `conf-emcc` package: this would need to be added as well.

Reply:
If anyone interested, for the JavaScript integration related issues, see https://gitlab.com/nomadic-labs/cryptography/ocaml-bls12-381/-/blob/ff6f2b599bbabd9a52ec5cf90c8eb13e3edac456/test/dune#L1.